### PR TITLE
 DataContext.run_validation_operator() now raises errors if no batches are passed or if they are of the wrong type.

### PR DESCRIPTION
**Pull request by Aylr**
Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For nontrivial changes, we will conduct both a design review and a code review. Please tag a core team member for code review, or leave blank and we will find someone for you!

- [x] Design Review (@jcampbell)
- [ ] Code Review

Thank you for submitting!

### CODE REVIEW

**Comment by jcampbell** [comment]
Design: GTG

File: docs/changelog.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ develop
   - Fix bullet list spacing issues
   - Fix 0.10. formatting
   - Drop roadmap_and_changelog.rst and move changelog.rst to the top level of the table of contents
-
+* DataContext.run_validation_operator() now raises DataContextErrors if no batches are passed or if they are the wrong type.
```
Comment (eugmandel) [inline docs/changelog.rst]:
```suggestion
* DataContext.run_validation_operator() now raises DataContextError if no batches are passed or if they are the wrong type.
```

**Review by eugmandel — APPROVED** [review]

**Comment by Aylr** [comment]
Required builds passed. Merging.

**Review by ayirplm — COMMENTED** [review]
LGTM

**Pull request merged**